core/domain/skill_domain_test.py

```diff
@@ -802,3 +802,22 @@ def test_augmented_skill_summary_gets_created(self):
         self.assertEqual(
             self.augmented_skill_summary.to_dict(),
             augmented_skill_summary_dict)
+
+
+class TopicAssignmentTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(TopicAssignmentTests, self).setUp()
+        self.skill_assignments = skill_domain.TopicAssignment(
```

Comment:
skill_assignments ...?

**Please proofread your whole PR and make sure that all the names are correct.** The overhead of these incremental changes is really not worth it. There are other places that have wrong names, too.

Reply:
I have fixed them now. 
Done.